Complete Core skeleton (#33)
datajoint-company/datajoint-docs: src/core/index.md

# Core

> DataJoint Core is a collection of open-source tools for building and operating scientific data pipelines.

Reviewer: For the sake of SEO, we should make sure to link back to glossary and between pages as much as possible.

```suggestion
DataJoint Core is a collection of open-source tools for building, operating, and visualizing scientific [data pipelines](./concepts/glossary#data-pipeline).
```

Reviewer: TBD in subsequent PR's.

## APIs

<div class="grid cards" markdown>

-   :fontawesome-brands-python:{ .lg .middle } **DataJoint for Python**

    ---

>     A Python client for defining, operating, and querying data pipelines.

Reviewer: Not sure I understand the purpose of mentioning Python here multiple times. It is in the title of the card. Why do we need to repeat this? Also, we should like to the glossary the usage of `data pipelines`.

Reviewer: TBD in subsequent PR's.

>     :octicons-arrow-right-24: [Documentation](https://docs.datajoint.org/python/) | [Source code](https://datajoint.com/datajoint/datajoint-matlab)

Reviewer 2: Below updating 
1. To point to new mkdocs
2. matlab -> python. I assume by 'source code', we mean github, right? 
```suggestion
    :octicons-arrow-right-24: [Documentation](https://datajoint.com/docs/core/datajoint-python/) | [Source code](https://github.com/datajoint/datajoint-python)
```

-   :fontawesome-brands-java:{ .lg .middle } **DataJoint for MATLAB**

    ---

>     A MATLAB client for defining, operating, and querying data pipelines.

Reviewer: Similar to previous comment.

Reviewer: TBD in subsequent PR's.

>     :octicons-arrow-right-24: [Documentation](https://docs.datajoint.org/matlab/) | [Source code](https://datajoint.com/datajoint/datajoint-matlab)

Reviewer 2: ```suggestion
    :octicons-arrow-right-24: [Documentation](https://datajoint.com/docs/core/datajoint-matlab) | [Source code](https://github.com/datajoint/datajoint-matlab)
```

-   :fontawesome-solid-flask:{ .lg .middle } **Pharus**

    ---

    A REST API server for interacting with DataJoint pipelines.

    :octicons-arrow-right-24: [Documentation](https://datajoint.github.io/pharus/) | [Source code](https://github.com/datajoint/pharus/)
 
</div>

> ## Web Applicaitons

Reviewer: ```suggestion
## Web Applications
```

<div class="grid cards" markdown>

-   :fontawesome-brands-chrome:{ .lg .middle } **LabBook**

    ---

	A browser-based graphical user interface for data entry and navigation. 

    :octicons-arrow-right-24: [Documentation](https://datajoint.github.io/datajoint-labbook/) | [Source code](https://github.com/datajoint/datajoint-labbook/)

-   :fontawesome-brands-chrome:{ .lg .middle } **SciViz**

    ---

>     A framework for making low-code web apps for data visualization.

Reviewer: ```suggestion
    A framework for making low-code web apps for [data pipelines](./concepts/glossary#data-pipeline).
```

Reviewer: TBD in subsequent PR's.

    :octicons-arrow-right-24: [Source code](https://github.com/datajoint/sci-viz)

</div>

> ## Docker Images

Reviewer: Docker is a particular technology that provides a solution for containerizing workloads. We may continue using Docker or change it in the future. The more appropriate term here is container.

```suggestion
## Container Images
```

  class dj,base,lab,hub,test,conda3,mysql boxes;
  classDef boxes stroke:#333; %% Grey stroke for class
```
<div class="grid cards" markdown>

-   :fontawesome-brands-docker:{ .lg .middle } [**datajoint/mysql**](https://hub.docker.com/r/datajoint/mysql)

    ---
    MySQL server pre-configured to work smoothly with DataJoint. 

> -   :fontawesome-brands-docker:{ .lg .middle } [**datajoint/miniconda3**](https://hub.docker.com/r/datajoint/niniconda3)

Reviewer: ```suggestion
-   :fontawesome-brands-docker:{ .lg .middle } [**datajoint/miniconda3**](https://hub.docker.com/r/datajoint/miniconda3)
```
